### PR TITLE
Add landing message. #1

### PR DESCRIPTION
**Pull request by azricicx**
Done bro !

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
     <h1>Hello, World!</h1>
     <p>This is a basic HTML page.</p>
+    <p>Saya Hebat !!</p>
```
Comment (azrimukminin13) [inline index.html]: Hebat lah sangat, ubah sekarang !

**Comment by azricicx** [comment]
sure can do !

**Review by azrimukminin13 — CHANGES_REQUESTED** [review]
Please, change the code as per my comment on the review

**Review by azrimukminin13 — APPROVED** [review]
Congratulation, youre very smart boy !

**Pull request merged**